LiteCore/Support/FilePath.cc

```diff
@@ -80,13 +81,29 @@ static int copyfile(const char* from, const char* to)
         errno = e;
         return write_fd;
     }
-    
-    if(sendfile(write_fd, read_fd, &offset, stat_buf.st_size) < 0) {
-        int e = errno;
-        close(read_fd);
-        close(write_fd);
-        errno = e;
-        return -1;
+
+    size_t expected = stat_buf.st_size;
+    ssize_t bytes = 0;
+    int noProgressRetry = 2;
+    while (bytes < expected) {
+        bytes = sendfile(write_fd, read_fd, &offset, expected - bytes);
```

Comment:
Shouldn’t this be += ?

Reply:
No, because I subtracted "exptected" by "bytes"

Reply:
Oh, actually, there is a problem as explain it.